hasjob/templates/detail.html

```diff
@@ -456,6 +449,23 @@
       return false;
     });
   });
+  $(function () {
+    var triggered_related = false;
+    $(window).scroll(function() {
+      if (!triggered_related &&
+          $(window).scrollTop() >= ($('#stickie-area').position().top - $(window).height()) &&
+          ($(window).height() + $(window).scrollTop()) < $(document).height()
+      ) {
+        triggered_related = true;
+        jQuery.get(
```

Comment:
Sure. I'll update. I used `jQuery` just because in the same file above this section `jQuery` is being used instead of `$`. https://github.com/iambibhas/hasjob/blob/0c81a6cac6f545b0cf20d5132880e451fa26b5cb/hasjob/templates/detail.html#L397